nulogy_graphql_api.gemspec

```diff
@@ -28,8 +28,8 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "graphql", "~> 2.0"
-  spec.add_dependency "graphql-schema_comparator", "~> 1.0"
+  spec.add_dependency "graphql", "~> 2.1"
+  spec.add_dependency "graphql-schema_comparator", "~> 1.2"
   spec.add_dependency "rails", ">= 6.0", "< 8.0"
```

Comment:
You could probably drop support for rails 6.0 by changing the specification to `spec.add_dependency "rails", ">= 6.1", "< 8.0"`